requirements/edx/post.txt

```diff
@@ -1,9 +1,12 @@
 # DON'T JUST ADD NEW DEPENDENCIES!!!
 #
 # If you open a pull request that adds a new dependency, you should notify:
-#   * @mollydb to check licensing
-#   * One of @e0d, @feanil, @fredsmith, @maxrothman, or @jibsheet
-#     to check system requirements
+#   * @mollydb - to check licensing
+#   * support@edx.org - to check system requirements
```

Comment:
I believe that only people in the `@edx/devops` team are able to mention it. Anyway, this is just copied from the header of `requirements/edx/base.txt`.